hash_practice/exercises.py

```diff
@@ -2,19 +2,43 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
+        Space Complexity: O(n)
     """

```

Comment:
👍  Your time complexity is correct, if the words are limited in length, like regular english words.  If the words could be any length then this is O(n * m log m) because you're sorting each word.